openassessment/xblock/test/test_staff_area.py

```diff
@@ -590,22 +574,40 @@ def test_cancel_submission_full_flow(self, xblock):
         bob_item["item_id"] = xblock.scope_ids.usage_id
         # Create a submission for Bob, and corresponding workflow.
         submission = self._create_submission(bob_item, {'text': "Bob Answer"}, ['peer'])
-
         incorrect_submission_uuid = 'abc'
         params = {"submission_uuid": incorrect_submission_uuid, "comments": "Inappropriate language."}
         # Raise flow not found exception.
-        resp = self.request(xblock, 'cancel_submission', json.dumps(params), response_format='json')
-        self.assertIn("Error finding workflow", resp['msg'])
-        self.assertEqual(False, resp['success'])
+        with self.assertRaisesMessage(SubmissionInternalError, '[u"\'abc\' is not a valid UUID."]'):
```

Comment:
Is it correct, that we check, that error has happened? Maybe we should investigate why this exception raised?

Reply:
Done